src/Core/Search/Discussion/Gambit/CreatedGambit.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Flarum\Core\Search\Discussion\Gambit;
+
+use Flarum\Core\Search\AbstractRegexGambit;
+use Flarum\Core\Search\AbstractSearch;
+use Flarum\Core\Search\Discussion\DiscussionSearch;
+use LogicException;
+
+class CreatedGambit extends AbstractRegexGambit
+{
+    /**
+     * http://stackoverflow.com/a/8270148/3158312
+     *
+     * @FIXME: Find better regular expression
```

Comment:
I firstly wanted [this](http://stackoverflow.com/questions/28020805/regex-validate-correct-iso8601-date-string-with-time), but with optional *parameters*. In that regex every part is required to pass.